CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to Mote, a Fedora-Infra project
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Mote, a Fedora-Infra project hosted by the Fedora community. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Fedora infrastructure](#fedora-infrastructure)
+  * [Fedora mailing lists](#mailing-lists)
+
+[How Can I be a part of the Community](#how-can-i-be-a-part-of-the-community)
+  * [Fedora badges](#fedora-badges)
+  * [Fedora Account System](#fas)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Mote Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to the Fedora community.
+
+## I don't want to read this whole thing I just have a question!!!
+
+Reach out to us on [telegram](https://t.me/joinfedora) or [IRC](https://fedoraproject.org/wiki/How_to_use_IRC). If you have a general question about the Fedora project, check out [the Fedora Wiki](https://fedoraproject.org/wiki/).
```

Comment:
I think it makes more sense to direct people to the CPE team for help instead of Fedora Join SIG. The people who have the rights to review and merge these Pull Requests are not very engaged in the Join SIG.